src/app/course-list/course/component.ts

```diff
@@ -35,11 +38,19 @@ export class CourseComponent {
     this.selectService.toggleCourse(course);
   }
 
+  public hasSelected() {
```

Comment:
The way we defined it in the past was that a course is "selected" iff it has >= 1 sections that are selected